test_files/abstract/abstract.js

```diff
@@ -2,7 +2,9 @@
  * @fileoverview added by tsickle
  * @suppress {checkTypes} checked by tsc
  */
-goog.module('test_files.abstract.abstract');var module = module || {id: 'test_files/abstract/abstract.js'};/**
+goog.module('test_files.abstract.abstract');
+var module = module || { id: 'test_files/abstract/abstract.ts' };
```

Comment:
Ah this is the test change.  That is good, it's been bugging me.  (Usual Evan complaint: would be easier to review if you made "add whitespace" a separate change from the one you're making here.)

Reply:
I cannot not make the whitespace change together with this. I'm emitting using the TS API, which then later on indents as it likes. The advantage is that it also fixes up the source map, so we no longer need to worry about that.